crates/l2/sequencer/l1_committer.rs

```diff
@@ -1182,14 +1159,27 @@ impl GenServer for L1Committer {
 
 /// Generate the blob bundle necessary for the EIP-4844 transaction.
 pub fn generate_blobs_bundle(
-    state_diff: &StateDiff,
+    blocks: &[Block],
+    fee_configs: &[FeeConfig],
```

Comment:
As a sanity check, we should eiter:
- Check `blocks` and `fee_configs` have same length. Or
- Receive a slice of tuples `&[(Block, FeeConfig)]`